ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -160,30 +181,38 @@ done
 	case "$ARCHITECTURE" in
 		"S390X" )
 			export MACHINE="s390-ccw-virtio"
-			export DRIVE="-drive file=$workFolder/${ARCHITECTURE}.dsk,if=none,id=hd0 -device virtio-blk-ccw,drive=hd0,id=virtio-disk0"
+			export DRIVE="-drive file=$workFolder/$OS.${ARCHITECTURE}.dsk,if=none,id=hd0 -device virtio-blk-ccw,drive=hd0,id=virtio-disk0"
 			export QEMUARCH="s390x"
 			export SSH_CMD="-net user,hostfwd=tcp::$PORTNO-:22 -net nic";;
 		"PPC64LE" )
 			export MACHINE="pseries-2.12"
-			export DRIVE="-hda $workFolder/${ARCHITECTURE}.dsk"
+			export DRIVE="-hda $workFolder/$OS.${ARCHITECTURE}.dsk"
 			export QEMUARCH="ppc64"
 			export SSH_CMD="-net user,hostfwd=tcp::$PORTNO-:22 -net nic";;
 		"AARCH64" )
-			export MACHINE="virt"
-			export DRIVE="-drive if=none,file=$workFolder/${ARCHITECTURE}.dsk,id=hd -device virtio-blk-device,drive=hd"
 			export QEMUARCH="aarch64"
-			export SSH_CMD="-device e1000,netdev=net0 -netdev user,id=net0,hostfwd=tcp:127.0.0.1:$PORTNO-:22"
-			export EXTRA_ARGS="-cpu cortex-a57 -bios /usr/share/qemu-efi-aarch64/QEMU_EFI.fd";;
+			case $OS in
+				"UBUNTU18" )
+					export MACHINE="virt,gic-version=max"
+					export DRIVE="-drive file=$workFolder/$OS.$ARCHITECTURE.dsk,if=none,id=drive0,cache=writeback -device virtio-blk,drive=drive0,bootindex=0"
+					export SSH_CMD="-netdev user,id=vnet,hostfwd=:127.0.0.1:$PORTNO-:22 -device virtio-net-pci,netdev=vnet"
+					export EXTRA_ARGS="-drive file=/qemu_base_images/arm64_tools/QEMU_EFI-flash.img,format=raw,if=pflash -drive file=/qemu_base_images/arm64_tools/flash1.img,format=raw,if=pflash -cpu max";;
+				"DEBIAN10" )
+					export MACHINE="virt"
+					export DRIVE="-drive if=none,file=$workFolder/$OS.${ARCHITECTURE}.dsk,id=hd -device virtio-blk-device,drive=hd"
+					export SSH_CMD="-device e1000,netdev=net0 -netdev user,id=net0,hostfwd=tcp:127.0.0.1:$PORTNO-:22"
+					export EXTRA_ARGS="-cpu cortex-a57 -bios /usr/share/qemu-efi-aarch64/QEMU_EFI.fd";;
+			esac ;;
 		"ARM32" )
 			export MACHINE="virt"
 			export QEMUARCH="arm"
 			export SSH_CMD="-device virtio-net-device,netdev=mynet -netdev user,id=mynet,hostfwd=tcp::$PORTNO-:22"
-			export DRIVE="-drive if=none,file=$workFolder/ARM32.dsk,format=qcow2,id=hd -device virtio-blk-device,drive=hd"
+			export DRIVE="-drive if=none,file=$workFolder/$OS.$ARCHITECTURE.dsk,format=qcow2,id=hd -device virtio-blk-device,drive=hd"
 			export EXTRA_ARGS="-kernel /qemu_base_images/arm32_tools/kernel.arm32 -initrd /qemu_base_images/arm32_tools/initrd.arm32 -append root=/dev/vda2";;
 		"RISCV" )
 			export QEMUARCH="riscv64"
 			export MACHINE="virt"
-			export DRIVE="-device virtio-blk-device,drive=hd -drive file=$workFolder/${ARCHITECTURE}.dsk,if=none,id=hd"
+			export DRIVE="-device virtio-blk-device,drive=hd -drive file=$workFolder/$OS.${ARCHITECTURE}.dsk,if=none,id=hd"
```

Comment:
For syntax consistency
```suggestion
			export DRIVE="-device virtio-blk-device,drive=hd -drive file=$workFolder/${OS}.${ARCHITECTURE}.dsk,if=none,id=hd"
```